src/trixi_include.jl

```diff
@@ -45,7 +45,12 @@ function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
     if !mpi_isparallel(Val{:MPIExt}())
         @info "You just called `trixi_include`. Julia may now compile the code, please be patient."
     end
-    Base.include(ex -> replace_assignments(insert_maxiters(ex); kwargs...), mod, elixir)
+    Base.include(ex -> replace_assignments(insert_maxiters(mapexpr(ex)); kwargs...),
+                 mod, elixir)
```

Comment:
```suggestion
    Base.include(replace_assignments ∘ insert_maxiters ∘ mapexpr; kwargs...),
                 mod, elixir)
```

Reply:
Well, the Trixi style guide says:
> Executable code should only use ASCII characters.

Reply:
@sloede? Composing functions is only available using `∘`. From my point of view, `∘` makes it easier to read the code in this case.

Reply:
Fine by me.

Reply:
Just realized it doesn't work like this because the kwargs are passed to `insert_maxiters`.